libraries/ms-common/inc/adc.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
Maybe I'm missing something, but where's the implementation of all these functions?